modules/bitgo/test/v2/unit/internal/tssUtils/ecdsaMPCv2/signTxRequest.ts

```diff
@@ -159,6 +189,28 @@ describe('signTxRequest:', function () {
     nockPromises[0].isDone().should.be.true();
     nockPromises[1].isDone().should.be.true();
     nockPromises[2].isDone().should.be.true();
+    // bitgoParty.endSession();
+  });
+
+  it('successfully signs a txRequest with a message for a dkls hot wallet with WP', async function () {
+    const nockPromises = [
+      await nockTxRequestResponseSignatureShareRoundOne(bitgoParty, txRequestForMessageSigning, bitgoGpgKey),
+      await nockTxRequestResponseSignatureShareRoundTwo(bitgoParty, txRequestForMessageSigning, bitgoGpgKey),
+      await nockTxRequestResponseSignatureShareRoundThree(txRequestForMessageSigning),
+      await nockSendTxRequest(txRequestForMessageSigning),
+    ];
+    await Promise.all(nockPromises);
+
+    const userShare = fs.readFileSync(shareFiles[vector.party1]);
+    const userPrvBase64 = Buffer.from(userShare).toString('base64');
+    await tssUtils.signTxRequest({
+      txRequest,
+      prv: userPrvBase64,
+      reqId,
+    });
+    // nockPromises[0].isDone().should.be.true();
+    // nockPromises[1].isDone().should.be.true();
+    // nockPromises[2].isDone().should.be.true();
```

Comment:
Are these needed still?

Reply:
Oh, I was gonna ask about this. Maybe there's a limitation on my knowledge of nocks or something, but as soon as I add the second test, these all evaluate to false for some reason, and I cannot figure out why.

Reply:
Yea we should check that the nock is done.